### PR TITLE
Add Input Component

### PR DESCRIPTION
**Pull request by varun13169**
Add Input Component
	- input html and css files.

### CODE REVIEW

File: components/input/input.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Delta UI | Input Component</title>
+    <link rel="stylesheet" href="/library/component.css">
+    <link rel="stylesheet"
+		href=
+"https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <style>
```
Comment (dcs-soni) [inline components/input/input.html:12]: I think you should avoid internal styling as much as possible. Any specific reason to apply internal style?

Everything else seems good to me 👍

**Review by zoef-yt — APPROVED** [review]
good to merge.

**Review by dcs-soni — CHANGES_REQUESTED** [review]

File: components/input/input.html (hunk 1 above)
Comment (varun13169) [reply components/input/input.html:12]: Hi @dcs-soni,
This is just the supporting code and will be changed while refactoring documentation.

Comment (dcs-soni) [reply components/input/input.html:12]: Alright then. It's g2g.

Comment (dcs-soni) [reply components/input/input.html:12]: Alright then. It's g2g.

**Review by dcs-soni — APPROVED** [review]

**Pull request merged**